src/module_PI/Raspberry_PI/main/PacketMessageTranslator.java

```diff
@@ -0,0 +1,110 @@
+package module_PI.Raspberry_PI.main;
+
+import java.util.*;
+
+public class PacketMessageTranslator {
+	
+	private Map<Integer, String> messageTable = new Hashtable<>();
+	private final int MAX_PACKET_DETAILS_INT = 55999;
+	private final int NUM_OF_BYTES_IN_PACKET_DETAILS = 3;
+	private final int NUM_OF_BYTES_IN_PACKET = 5;
+	private final int MAX_DETAILS_ONLY_INT = 999;
+	private final int MIN_FULL_PACKET_INT = 11001;
+	
+	public PacketMessageTranslator(){
+		buildTable();
+	}
+	
+	private void buildTable(){
+		messageTable.put(1, "Execute Command: Arm");
```

Comment:
If tests require an external dependency then they aren't really "Unit Tests". If it relies on an external dependency like that then it isn't a "Unit Test". See my comment on the previous pull request https://github.com/Lastrellik/Calico-Home-Security/pull/69 and specifically look at the comments I made here: https://github.com/Lastrellik/Calico-Home-Security/pull/69/files/825afa15f6f81b6afadf381b11622568dc45f5f8#diff-5421dd2a9b2dcecfc8aafd309c0c8faa

Essentially you should 'mock' the response an Arduino would be sending.